### PR TITLE
Adding tests for get_mentors_api

### LINKED ISSUE #22: Test Get Mentors API
**Is your feature request related to a problem? Please describe.**
We need to test the Get Mentors API refactored in #15

**Describe the solution you'd like**
Relevant tests must be added to tests/mentors/tests_get_mentors_api.py

**Additional context**
Few suggestions for how the tests could be are as follows:
* Invalid Input (e.g. `twentynineteen`)
* Fetch empty mentor list
* Fetch list with multiple mentors
* etc.

### PR DESCRIPTION
**Pull request by akhil-maker**
### Description
Adding test for get_mentors_api. It includes tests

               -  to get single mentor for a particular year.
               - to get multiple mentors for a year and not allowing mentor with flag=False
               - to get an empty list of mentor as there will be no mentors added for that year
               - for invalid input

**Type of Change:** (Delete irrelevant options)
- Testing

**Code/Quality Assurance Only** (Delete irrelevant options)

- Bug fix (non-breaking change which fixes an issue)
-Testing for an api

### How Has This Been Tested?
Run locally and tested.

### Checklist:
- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] Any dependent changes have been merged
- [ ] Update Swagger documentation and the exported file at /docs folder
- [ ] Update requirements.txt

**Code/Quality Assurance**
- [x] My changes generate no new warnings 
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been published in downstream modules

### CODE REVIEW

**Comment by akhil-maker** [comment]
@anshsrtv and @priyanshisharma 
please give your reviews.

**Review by priyanshisharma — CHANGES_REQUESTED** [review]
Looks great to me 🎉 
Add an `__init__.py` file to the `tests.mentors` so that python identifies it as a module.

**Review by anshsrtv — COMMENTED** [review]
You can also check the length of the array returned in the responses, like in case of `test_pass_get_mentors_empty`, you can test this.

`self.assertEqual(len(response.data['data']), 0)`

>

File: tests/mentors/test_get_mentors.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+    def test_pass_get_mentors_single(self):
+        """
+        Test for single mentor
+        """
+        get_group_api = "/mentors/list/2020/"
+        get_client = APIClient()
+        response = get_client.get(get_group_api)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
```
Comment (sahil9001) [inline tests/mentors/test_get_mentors.py:68]: What's the point of testing single mentor list if we are already testing multiple mentor test? @akhil-maker

Comment (akhil-maker) [reply tests/mentors/test_get_mentors.py:68]: Actually I thought if I am adding test for multiple and empty, then why not for single one of a different year. So, added.

Comment (sahil9001) [reply tests/mentors/test_get_mentors.py]: Um.. I think it's pretty redundant because you are testing for multiple users in the year 2019 and no users for 2021, so I don't think this is needed here. Any suggestions @anshsrtv sir?

Comment (akhil-maker) [reply tests/mentors/test_get_mentors.py]: Okay! Let's see the suggestions from sir.

Comment (anshsrtv) [reply tests/mentors/test_get_mentors.py]: @sahil9001 With the recent commits, it does make sense now, and test cases can be redundant, keeping in mind, the objectives of testing. ✌️ 

@akhil-maker please check the status codes, as well, in all the test cases.🙌

Comment (sahil9001) [reply tests/mentors/test_get_mentors.py]: Okay then !

Comment (akhil-maker) [reply tests/mentors/test_get_mentors.py]: @anshsrtv I will add the code to check status codes soon.

**Review by priyanshisharma — APPROVED** [review]
Works good for me 🎉

**Review by anshsrtv — APPROVED** [review]
